src/AWS.Deploy.Recipes/CdkTemplates/AspNetAppAppRunner/Configurations/Configuration.cs

```diff
@@ -0,0 +1,120 @@
+// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
+// SPDX-License-Identifier: Apache-2.0
+
+namespace AspNetAppAppRunner.Configurations
+{
+    public class Configuration
+    {
+        /// <summary>
+        /// The Identity and Access Management Role that provides AWS credentials to the application to access AWS services
+        /// </summary>
+        public IAMRoleConfiguration ApplicationIAMRole { get; set; }
+
+        /// <summary>
+        /// The Identity and Access Management (IAM) role that provides gives the AWS App Runner service access to pull the container image from ECR.
```

Comment:
change "provides gives" to either "provides" or "gives"

Reply:
Fixed